x/dex/keeper/limit_order_tranche.go

```diff
@@ -55,7 +55,7 @@ func (k Keeper) SaveTranche(ctx sdk.Context, tranche *types.LimitOrderTranche) {
 	if tranche.HasTokenIn() {
 		k.SetLimitOrderTranche(ctx, tranche)
 	} else {
-		k.SetInactiveLimitOrderTranche(ctx, tranche)
+		k.SaveInactiveTranche(ctx, tranche)
```

Comment:
It's counterintuitive that PR is about removing of old tranches, but the method name implies to save them instead.

Is there any way to change it?